scripts/tasks/dice_roll/task.md

```diff
@@ -0,0 +1,136 @@
+# Dice roll
+
+**Difficulty**: EASY - AVERAGE  
+**Linux related topics**: Text processing and filters
+
+---
+## Task:
+Create a script which simulates dice roll with different types and numbers of dice.  
+Follow tasks from number 1 to slowly add functionality into your code.  
+
+For quick reference to specific die we will use NdS notation where N is number of dice  
+and S is number of die sides.  
+e.g. 2d8 means 2 dice with 8 sides
+
+&nbsp;
+
+---
+**EASY:**
+
+**1) Single 1d6**  
+Create the most basic implementation of one die roll. Roll 1d6.
+
+```
+./roll_dice.sh  
+5
+```
+&nbsp;
+
+**2) Multiple 1d6**  
+Make your script roll Nd6 based on your input.
+
+```
+./roll_dice.sh 2  
+3  
+1  
+```
+&nbsp;
+
+**3) Single 1dN. Use die reference in input.**  
+Make your script roll single 1dN based on your input.
+
+```
+./roll_dice.sh 1d12  
+11
+```
+&nbsp;
+
+**4) Multiple 1dN.**  
+Make your script roll single NdN based on your input.
+
+```
+./roll_dice.sh 3d12  
+11  
+7  
+2
+```
+&nbsp;
+
+---
+**AVERAGE:**
+
+**5) Various combinations of NdN.**  
+Make your script roll single NdN based on your input.
+
+```
+./roll_dice.sh 1d12 2d4  
+11  
+3  
+2
+```
+&nbsp;
+
+**6) Add some pretty output**  
+Make your script roll single NdN based on your input. You are not limited in your creativity, add any type of output you like.  
```

Comment:
You're only limited by your imagination here, add some pretty output to your roll results :) .